CHANGELOG.md

```diff
@@ -2,6 +2,53 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.20 - May 20, 2021
+## Breaking changes ⚠️
+* Introduce ResourceOptionsManager to configure the default resource options, and removed the xml configuration options for cache path and tile store path. ([#339](https://github.com/mapbox/mapbox-maps-android/pull/339))
+* Rename ambient cache database to ambient_cache.db  ([#314](https://github.com/mapbox/mapbox-maps-android/pull/314))
+* Remove usage of asset path from the codebase, as it is not used in Android SDK. ([#334](https://github.com/mapbox/mapbox-maps-android/pull/334))
+* Parsing geojson on a worker thread.. ([#327](https://github.com/mapbox/mapbox-maps-android/pull/327))
```

Comment:
I would include more information here in changelog as it is pretty important.

Reply:
Can you provide detailed information on this? I just copied from #327